cicecore/cicedyn/analysis/ice_history_shared.F90

```diff
@@ -81,6 +81,8 @@ module ice_history_shared
           real (kind=dbl_kind) :: conb    ! additive conversion factor
           character (len=1) :: vhistfreq  ! frequency of history output
           integer (kind=int_kind) :: vhistfreq_n ! number of vhistfreq intervals
+          logical (kind=log_kind) :: avg_ice_present = .false. ! only average where ice is present
+          logical (kind=log_kind) :: mask_ice_free_points  = .false. ! mask ice-free points
```

Comment:
Is it useful/necessary to initialize the flags here?  No other ice_hist_field variables are initialized here, and they will be initialized in define_hist_field if needed.  Think it might be more consistent and still robust to not initialize here.

Reply:
I agree that since the default values are set using the local variables `l_avg_ice_present` and `L_mask_ice_free_points` in define_hist_field, there is no absolute need to initialize them here.

I can remove that for consistency.

Reply:
Thanks @phil-blain.

Reply:
Done and force-pushed.